### DISCUSSION TITLE
how to build a hermes.so for Android platform

### DISCUSSION THREAD

**Discussion by zhuozp** [facebook/hermes#1083] [Q&A]
can anyone tell me how to build a hermes.so. (for we want to have a custom building)
I try to read the doc about https://hermesengine.dev/docs/cross-compilation. 
but seems hard to understand the 2nd step.

**Comment by zhuozp** [comment]
has build the hermes.so successfully

**Reply by sujankh** [reply]
@zhuozp i also got confused in the 2nd step to cross-compile for Android platform. If possible, would you mind sharing the steps you took? I am confused with where/how to execute the android/gradlew. Thank you.
